lockbox-iosTests/SettingsPresenterSpec.swift

```diff
@@ -31,9 +31,11 @@ class SettingsPresenterSpec: QuickSpec {
 
     class FakeRouteActionHandler: RouteActionHandler {
         var routeActionArgument: RouteAction?
+        var invokeWasCalled = false
 
         override func invoke(_ action: RouteAction) {
             self.routeActionArgument = action
+            self.invokeWasCalled = true
```

Comment:
nit: this is extraneous, you can just check that the routeActionArgument isn't nil